recipes/partialjson/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "partialjson" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91217e19a15049332df534477f56420065ad1729cedee7d8c7433e1d2acc7dca
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python 3.6
+    - pip
```

Comment:
I'm surprised this is working without `setuptools` under `host`, but it may just be because older versions of python/pip automatically brought it along as a dependency. It's probably good to add it explicitly though.